definitions/npm/react-jss_v6.1.x/flow_v0.30.x-/react-jss_v6.1.x.js

```diff
@@ -8,6 +8,8 @@ declare module 'react-jss' {
 
   declare type Null = null | void;
 
+  declare type Dictionary = { [key: string]: {} }
```

Comment:
According to https://github.com/flowtype/flow-typed/issues/16 there is no way of specifying the dependencies for libdefs in `flow-typed`, but I might be wrong somewhere since I am new to this project as well. Maybe @marudor could clarify it to us?